src/Razor/test/testapps/MvcWithComponents/Views/Shared/NavMenu.razor

```diff
@@ -1,2 +1,3 @@
+@Test.Foo
```

Comment:
Hmm, are you not able to make this edit in the test execution? If not i'd leave a big comment here so no one removes this in the future.

Reply:
Tried it but that made the tests more flaky. Will add a comment.

Reply:
Oh yes I remember the actual problem with that. I had to save the file in order for `Test.cs` to see it as a reference which makes the test too complicated.